packages/block-library/src/site-logo/block.json

```diff
@@ -33,6 +33,19 @@
 		"html": false,
 		"align": true,
 		"alignWide": false,
+		"__experimentalBorder": {
+			"color": true,
+			"radius": true,
+			"width": true,
+			"style": true,
+			"__experimentalSkipSerialization": true,
+			"__experimentalDefaultControls": {
+				"color": true,
+				"radius": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
```suggestion
```

Recent feedback has been that if a block doesn't display its spacing controls by default then we should also make the border controls optional.